data/BaseStore.js

```diff
@@ -56,6 +56,12 @@ export class BaseStore {
     /** Is this store empty before filters have been applied? */
     get allEmpty() {return this.allCount === 0}
 
+    /** Do any post-filtered records have children? */
```

Comment:
I would lose these getters for now.  The names don't seem intuitive to me (might have used the word 'is') and they don't add a lot of value.  Just do the 'some' check in the grid.  

Also, think we should just always look at pre-filtered records.  Seems weird that the indentation might change just by tweaking a local filter (e.g. StoreFilterField)